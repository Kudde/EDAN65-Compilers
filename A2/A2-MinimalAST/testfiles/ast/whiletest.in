int whiletest() {
while(1) {
x = 2;
}
}
