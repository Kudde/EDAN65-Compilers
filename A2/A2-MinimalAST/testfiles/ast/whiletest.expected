Program
  List
    Fun
      TypeDecl Type="int"
      IdDecl Decl="whiletest"
      Param
        List
      Block
        List
          WhileStmt
            Literal NUMERAL="1"
            Block
              List
                AssignStmt
                  ID ID="x"
                  Literal NUMERAL="2"
